routes/api/stickers.js

```diff
@@ -96,6 +97,20 @@ router.get('/obtain/:eventId', async (req, res) => {
   }
 });
 
+//endpoint para buscar cromos por su id
+router.get('/:stickerId',isAdmin, async (req,res)=>{
+  if (isNaN(req.params.stickerId)){
```

Comment:
```suggestion
  if (Number.isNaN(req.params.stickerId)){
```
Lo mismo que en events